src/wovn_interceptor.php

```diff
@@ -43,7 +45,10 @@
     }
     // use the callback of ob_start to modify the content and return
     ob_start(function ($buffer) use ($headers, $store, $diagnostics, $benchmarkStart) {
-
+        if ($headers->shouldRedirect()) {
+            // this carries an implied HTTP 302
+            header("Location: " . $headers->computeRedirectUrl());
```

Comment:
https://stackoverflow.com/questions/3553698/php-should-i-call-exit-after-calling-location-header my assumption is we should exit() after such a redirect.

Reply:
I also thought there has to be some kind of termination there. I tried `fastcgi_finish_request` but it's not working. That's kinda also the reason why I did the redirect in the header too, to fix the behavior.

Reply:
I guess even the redirect we already do did not do this. So maybe we don't need to.